src/webui/www/private/scripts/dynamicTable.js

```diff
@@ -1206,23 +1206,23 @@ window.qBittorrent.DynamicTable = (function() {
 
             switch (filterName) {
                 case 'downloading':
-                    if (state != 'downloading' && !~state.indexOf('DL'))
+                    if (state != 'downloading' && state.indexOf('DL') === -1)
```

Comment:
I wonder if the left condition can be `state != 'downloading'`

Reply:
If that doesn't break the UI, then you are encouraged to change it.